package.json

```diff
@@ -2,24 +2,25 @@
   "name": "root",
   "private": true,
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "samples/react-app"
   ],
   "scripts": {
-    "build": "npm run prettier:check && npm run clean && lerna run build",
+    "build": "npm run prettier:check && npm run clean && lerna run build --scope @microsoft/*",
     "build:mgt": "cd ./packages/mgt && npm run build",
     "build:mgt-element": "cd ./packages/mgt-element && npm run build",
```

Comment:
TBH, I'm concerned that is going to get out of hand with scripts in the root if we start adding a build command for every project. If there is a real benefit, I'm all for it, but otherwise, I'd stick with simplicity.

FWIW, you can run any script in any project with lerna run (for example for mgt: `lerna run build --scope @microsoft/mgt`)

Reply:
Personally, I love the granularity. It's a one time thing per project, and dramatically less characters

Reply:
Done